contrib/kind-ha.yaml

```diff
@@ -18,6 +18,13 @@ nodes:
    extraMounts:
      - hostPath: /tmp/kind
        containerPath: /var/run/secrets/kubernetes.io/serviceaccount/
+   kubeadmConfigPatches:
```

Comment:
Is this change from L21 to L27 not needed for the other control-plane nodes at L28 and L32? Thanks